public/js/angularjs-all.min.js

```diff
@@ -1,13 +1,13 @@
 /*! insight-ui-btcz 0.4.0 */
```

Comment:
Was there a particular reason for modifying this file?

Reply:
I didn't actually modify it, not directly, but I think the compile somehow caused a subtle difference. And I didn't change the version of Angular or anything either, just did the bower install.

The files I changed directly were the index.html, a couple of the view files, one of the app JS files and the common CSS.

Reply:
Can you please run `./node_modules/.bin/grunt compile` and then push again? I can't reproduce the build and the blob of code is not reviewable, so unfortunately, I can't approve yet.

Reply:
Hey, I ran the compile again but there was no change so nothing to push. I reverted the file back to the original version before I pushed all my changes. Hopefully that resolves things.